python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1782,6 +1782,15 @@ def test_parquet_writer_statistics(tmpdir, pdf):
     if "col_category" in pdf.columns:
         pdf = pdf.drop(columns=["col_category", "col_bool"])
 
+    timedelta_types = [
```

Comment:
@devavret  should we add duration types to the pdf fixture?

Reply:
When we talked, the idea was to add these types to the fixture first and hope no other test fails. If they do then make this a local change to the stats test to unblock this PR and file the breakages separately. I suppose @kaatish is going to reveal the tests that broke.

Reply:
@devavret Yes, that was my experience. Adding timedelta types to the build_pdf function causes tests to fail.

Reply:
Will open an issue to add duration coverage, and we can go ahead and merge this one as-is. Objections can be filed until CI passes :)